sdkgen/nodejs-request/lib/util.js

```diff
@@ -0,0 +1,81 @@
+const codegen = require('postman-code-generators'),
+  sdk = require('postman-collection');
+
+/**
+ * [Description]
+ *
+ * @param {String} requestSnippet - Request snipept generated by postman-code-generator
+ * @returns {String} - Request snippet string with replaced collection variables
+ */
+function replaceVariables (requestSnippet) {
+  var variableDeclarations = requestSnippet.match(/{{.*}}/g);
```

Comment:
Please check this regex. It will consider everything between two `{{`, `}}`. And will fail if there are multiple variables
https://regex101.com/r/NfZ74n/1

Reply:
/{{[^{{]*}}/g  this regex should work and is working rn, looking for more edge cases which may errors